### PR TITLE
fix consensus cross chain msg compatible old version

### PR DESCRIPTION
**Pull request by tanZiWen**

### CODE REVIEW

File: consensus/vbft/types.go [hunk 1]
```diff
@@ -135,8 +135,8 @@ func (blk *Block) Deserialize(data []byte) error {
 	}
 
 	var crossChainMsg *types.CrossChainMsg
-	hasEmptyCCM, irr, eof := source.NextBool()
-	if irr || eof {
+	hasEmptyCCM, irr, _ := source.NextBool()
```
Comment (laizy) [inline consensus/vbft/types.go:139]: document why not handle eof here

Comment (tanZiWen) [reply consensus/vbft/types.go:139]: added

File: consensus/vbft/block_pool.go [hunk 2]
```diff
@@ -477,6 +477,7 @@ func (pool *BlockPool) newBlockCommitment(msg *blockCommitMsg) error {
 		if crossChainMsgSig, present := msg.CrossChainMsgEndorserSig[endorser]; present {
```
Comment (Honglei-Cong) [inline consensus/vbft/block_pool.go]: check if CrossChainMsgEndorserSig is nil?

Comment (tanZiWen) [reply consensus/vbft/block_pool.go]: msg_builder // 383 had made CrossChainMsgEndorserSig

**Review by Honglei-Cong — APPROVED** [review]

**Pull request merged**